### ISSUE TITLE
[Z80 Assembly feature] Allow temporary labels (starting with a backtick, ` )

### ISSUE THREAD

**Issue by Dotneteer** [Dotneteer/spectnetide#119]
Temporary labels have a limited scope, only between persistent labels:

```
SetPixels:        ; Persistent label
  ld hl, #4000
  ld a,#AA
  ld b,#20
`loop:            ; Temporary label (scope #1)
  ld (hl),a
  inc hl
  djnz `loop
SetAttr:          ; Persistent label, scope #1 disposed here
  ld hl,#5800
  ld a,#32
  ld b,#20
`loop:            ; Temporary label (scope #2)
  ld (hl),a
  inc hl
  djnz `loop
  ret

; scope #2 still lives here
; ...
Another: ; Persistent label, scope #2 disposed here
  ld a,b
```

**Comment by Dotneteer** [comment]
Implemented, will be released soon.

**Comment by Dotneteer** [comment]
Released in v1.17.0
